### ISSUE TITLE
Add a logo and link to http://diversitycharter.org

### ISSUE THREAD

**Issue by mattwynne** [cucumber-attic/website#149]
Diversity Charter is a nice thing being organised by my new friend @ewebber and I'd like to support it by putting a logo and link somewhere on our site.

First question - @romaingweb and @theo-england where do you think the most appropriate place is? I'd suggest in the overall site footer, but I'm totally open to suggestions.

**Comment by mattwynne** [comment]
Logos here: http://diversitycharter.org/logo-download/

**Comment by theo-england** [comment]
Yeah site footer sounds good. I reckon we include it on the CukeUp! pages too.
